scripts/AbstractWidget.py

```diff
@@ -0,0 +1,322 @@
+import re
+import sys
+import os
+class eventHandlerInfo:
+    def __init__(self, java_src, identity, java_line_number,xml_line_number = None, xml_src = None, name=None):
+        self.xml_src = xml_src
+        self.java_src = java_src
+        self.identity = identity
+        self.java_line_number = java_line_number
+        self.xml_line_number = xml_line_number
+        self.name = name
+
+    def toString(self):
```

Comment:
Typically people override the standard Python interface `__str__`https://stackoverflow.com/questions/4912852/how-do-i-change-the-string-representation-of-a-python-class